src/Resources/ResourceManager/Implementation/Policy/NewAzurePolicyAssignment.cs

```diff
@@ -44,7 +44,7 @@ public class NewAzurePolicyAssignmentCmdlet : PolicyCmdletBase, IDynamicParamete
         /// <summary>
         /// Gets or sets the policy assignment scope parameter
         /// </summary>
-        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.NewPolicyAssignmentScopeHelp)]
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.NewPolicyAssignmentScopeHelp)]
```

Comment:
I suggest update these help messages to inform the user that `Scope` now defaults to what